Packs/PlaybooksUsingScripts/Scripts/PlaybooksUsingScripts/PlaybooksUsingScripts.py

```diff
@@ -0,0 +1,98 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def get_custom_scripts_playbooks():
+    results = {}
+    final_results = []
+    req_response = demisto.executeCommand(
+        "demisto-api-post", {"uri": "automation/search", "body": "{\"query\":\"system:F AND hidden:F AND deprecated:F\"}"})
+    list_scripts = req_response[0]['Contents']['response']['scripts']
+    if list_scripts == None:
+        return_error("No custom scripts found.")
+    else:
+        for item in list_scripts:
+            results['ScriptName'] = item['name']
+            results['Type'] = item['type']
+            results['User'] = item['user']
+            results['Modified'] = item['modified']
+            query = {"query": "script.name:" + str(item['name']) + " AND hidden:F AND deprecated:F"}
+            playbooksUsingScripts = demisto.executeCommand(
+                "demisto-api-post", {"uri": "playbook/search", "body": query})[0]['Contents']['response']['playbooks']
+            if playbooksUsingScripts != None:
+                pbnamelist = []
+                for item in playbooksUsingScripts:
+                    pbnamelist.append(item['name'])
+                results['playbooksUsingScript'] = pbnamelist
+            else:
+                results['playbooksUsingScript'] = "Not used in any playbooks"
+            final_results.append(results.copy())
+        markdown = ''
+        markdown += tableToMarkdown('Custom scripts Used in Playbooks', final_results)
```

Comment:
This lie is not inside the for loop so it is identical:

```suggestion
        markdown = tableToMarkdown('Custom scripts Used in Playbooks', final_results)
```